specs/appwindow-spec.md

```diff
@@ -0,0 +1,68 @@
+Window.AppWindow api
+===
+New api to simplify accessing [appwindow](https://docs.microsoft.com/en-us/windows/apps/windows-app-sdk/windowing/windowing-overview) functionality through WinUI code
+
+# Background
+AppWindow provides good set of functionalities for using HWNDs than dealing with win32 apis directly. It is already accessible to winui desktop apps via winappsdk runtime. Currently, users must write a lot of boiler plate code to get to appwindow object and use it. 
```

Comment:
Could you manually wrap at ~100 columns? Not all editors can handle soft returns.